boards/st/nucleo_u5a5zj_q/nucleo_u5a5zj_q-common.dtsi

```diff
@@ -55,22 +55,30 @@
 	status = "okay";
 };
 
+/* This board has a 16MHz crystal attached */
+&clk_hse {
+	clock-frequency = <DT_FREQ_M(16)>;
+	status = "okay";
+};
+
 &clk_lse {
 	status = "okay";
 };
 
+/* Note: this is configured as 4MHz source clock */
 &clk_msis {
 	status = "okay";
 	msi-range = <4>;
 	msi-pll-mode;
 };
```

Comment:
Do we wan to keep MSIS and HSE both enabled? Since HSE will now source PLL, maybe it is a good idea to remove MSIS node from here? (like I did in https://github.com/zephyrproject-rtos/zephyr/pull/81774/commits/af145ce8c674759e3e3e660fe05943092cbd9838)

Reply:
I don't think there is support for clock-down in Zephyr right now.

@erwango what is your preference about keeping or removing `clk_msis` after we switch to HSE by default?

Reply:
> @erwango what is your preference about keeping or removing clk_msis after we switch to HSE by default?

If not used, then it should be removed indeed.